File: kyu_8/038_Calculate_BMI/python/main.py
# Solution 1
def bmi(weight, height):
    result = weight / height ** 2

    if result <= 18.5:
        return "Underweight"
    elif result <= 25.0:
        return "Normal"
    elif result <= 30.0:
        return "Overweight"
    return "Obese"

print(bmi(80, 1.8)) # Normal

File: kyu_8/038_Calculate_BMI/javascript/js/main.js
// solution 1
function bmi(weight, height) {
  result = weight / height ** 2;

  if (result <= 18.5) return "Underweight";
  else if (result <= 25.0) return "Normal";
  else if (result <= 30) return "Overweight";
  return "Obese";
}

console.log(bmi(80, 1.8)); // Normal

